lib/email.js

```diff
@@ -6,6 +6,7 @@ var fs = require('fs');
 var contents = fs.readFileSync(__dirname+ '/email.html');
 var contents_txt = fs.readFileSync(__dirname+ '/email.txt', 'utf8');
 var email     = require('emailjs');
+var dns = require('dns');
```

Comment:
Isn't this requiring Node's "dns" module rather than the "dns" module specified in package.json below? It looks like that other module is a DNS server https://github.com/hbouvier/dns